File: demo/ChengFenStore/Models/Order.cs
namespace ChengFenStore.Models
{
    public class Order
    {
        public int OrderId { get; set; }
        public int UserId { get; set; }
        public string ProductDetails { get; set; }
        public string DeliveryMethod { get; set; }
        public string OrderStatus { get; set; }
        public List<OrderStatus> OrderStatusHistory { get; set; }

        public Order()
        {
            OrderStatusHistory = new List<OrderStatus>();
        }
    }
}

File: demo/ChengFenStore/Models/Order.py
from ..Data.AppDbContext import db

class Order(db.Model):
    order_id = db.Column(db.Integer, primary_key=True)
    user_id = db.Column(db.String(20), db.ForeignKey('user.phone_number'), nullable=False)
    product_details = db.Column(db.String(200), nullable=False)
    delivery_method = db.Column(db.String(50), nullable=False)
    order_status = db.Column(db.String(50), nullable=False)
    order_status_history = db.relationship('OrderStatus', backref='order', lazy=True)
